proto/rill/runtime/v1/api.proto

```diff
@@ -348,9 +348,7 @@ message Connector {
   // Name of the connector
   string name = 2;
   // Config for the connector
-  map<string, string> config = 3;
```

Comment:
I think the `runtimev1.Connector` type is persisted in the database, so I'm not sure this change is safe. Can you check that? https://github.com/rilldata/rill/blob/c92a41f03b205dad72e765ef979d8a848a685a3f/runtime/drivers/sqlite/registry.go#L97-L100

Maybe keep the name here `config`. Also, are there any other places you think compatibility could be a problem?